galore/cross_sections.py

```diff
@@ -281,3 +281,147 @@ def _eval_fit(energy, coeffs):
                 orb: _eval_fit(energy, np_fromstr(coeffs))
                 for orb, coeffs in orbitals_fits}})
     return el_cross_sections
+
+
+
+import tarfile
+import numpy as np
+def read_csv_file(tar_file_name,file_path):
+    '''read csv file 
+    Input:  the file name
+    Output: main matrix of each file'''
+    
+    ###Open zipfile
+    with tarfile.open(tar_file_name) as tf:
+        with tf.extractfile(file_path) as hello:
+            data = hello.read().decode()
+    a = data.split('\r\n')
```

Comment:
Use descriptive names for variables. It is hard to read a line of code operating on `a`, `b`, `c`, and `d` and understand what it is supposed to be doing.

Reply:
The new name `data_string` is a bit better because it is at least "greppable". But it's also a bit misleading because `data_string` is not actually a string, it's a list. Maybe something like `data_lines` would be better, as this conveys how it was split?

Reply:
`data_strings` would at least be better. If you read `data_string[0]` it looks like it indexes a single letter from a string. Whereas `data_strings[0]` clearly gets a longer string, which can be split.